sql/SQLCode.sql

```diff
@@ -0,0 +1,23 @@
+CREATE User(
+ID INT AUTO_INCREMENT PRIMARY KEY,
+u_name VARCHAR(30) NOT NULL,
+u_pwd VARCHAR(30) NOT NULL,
+is_admin BIT DEFAULT 0,
+);
+
+CREATE MetaData(
+ID INT AUTO_INCREMENT PRIMARY KEY,
+file_size DECIMAL(4,0) NOT NULL,
+date_added INT,
+number_of_updates SMALLINT, 
+user_ID VARCHAR(5),
+file_ID VARCHAR(5),
```

Comment:
Has to be the same type as id of the other table I think
```suggestion
file_ID INT,
```